Codecademy/gamut
# feat(Illustrations): Add computer lock illustration

### Overview


Adds new computer + lock illustration


### PR Checklist

- [x] Related to designs: [Figma](https://www.figma.com/file/oQb1a4kuzvlI925Oi0LgPj/Free-trial-onboarding---Guided-exploration?node-id=1593%3A24844&mode=dev)
- [x] Related to JIRA ticket: [FLOW-514]

#### Testing Instructions



1. Go to [preview](https://64934ad12547ba06ef15a5f6--gamut-preview.netlify.app/?path=/docs/atoms-illustrations--editable-illustration)
2. Check the illustration!

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |




[FLOW-514]: https://codecademy.atlassian.net/browse/FLOW-514?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
